src/bindings/nodejs/index.d.ts

```diff
@@ -0,0 +1,81 @@
+export type ValhallaConfig = Record<string, any>;
+export type Query = Record<string, any>;
+export type Response = Record<string, any>;
+
+export class Actor {
```

Comment:
This is needed to also be more friendly when used from TypeScript. We could thoroughly describe all types here, but I decided to avoid doing so: high risk it will go out of sync with the rest of repo, so generic `Record<string, any>` seems to be enough